CMakeLists.txt

```diff
@@ -277,13 +277,16 @@ conan_check(VERSION 1.0.0 REQUIRED)
 
 conan_add_remote(NAME bincrafters
   URL https://api.bintray.com/conan/bincrafters/public-conan)
+conan_add_remote(NAME lefticus
+  URL https://api.bintray.com/conan/lefticus/nrel)
```

Comment:
@macumber primarily for the Ruby for sure. In some cases you might need to create a temporary fork of a given package too though, which currently I tend to push to my jmarrec/testing bintray.  That's because bincrafters tend to be slow to propragate changes upstream (due to the very nature of bincrafters... they are like a gazillion individual repos).

In any case, once the bintray remote is setup, exporting just conan-openstudio-ruby or exporting that + a ton of others would look exactly the same.

Would need to create an organization (rather than a single user) on bintray for NREL, and adding a couple of people (myself included) to it so they can push stuff there.